### DISCUSSION TITLE
liblash-compat

### DISCUSSION THREAD

**Discussion by ondinegreen** [amsynth/amsynth#192] [General]
Complete ignoramus with programming here (can only just compile for source), so not sure where to put this, apologies if it's the wrong place:

Just letting you know that amsynth no longer seems to work out-of-the-box on the latest Ubuntu/Ubuntu Studio (20.04.1 LTS), because the LASH compatibility package (liblash-compat-*) is no longer included in that distribution. I got it working by using the version from the 18.04 distribution, but Ubuntu doesn't like you doing things like that, lol.

Anyway, just thought someone here might decide this was useful information.

**Comment by nickdowell** [comment]
Hi @ondinegreen thanks for sharing.

When you say "amsynth no longer seems to work out-of-the-box" do you mean it fails to run at all, or that it doesn't contain LASH support?

**Reply by ondinegreen** [reply]
What I mean is that it just does not function. It does not start. When I run it in terminal I get an error message along the lines of "could not find package liblash.so.1debian0" (sorry I didn't capture the precise words)

**Reply by nickdowell** [reply]
I installed amsynth on a fresh install of Ubuntu 20.04 via `sudo apt install amsynth` and found that it runs fine, no liblash related issues.

Did you install amsynth from another source?
